### PR TITLE
Add domain field for media.net only

### PR DESCRIPTION
**Pull request by epechuzal**
Added logic to add the domain for media.net in that specific case.
Later, all publishers will have a `domain` field in the db so this logic will be streamlined.

[Story](https://www.pivotaltracker.com/story/show/167204412)

### CODE REVIEW

File: src/sellers.js [hunk 1]
```diff
@@ -16,14 +16,18 @@ const fetchPubs = async () => {
...
       if (error) throw error;
       resolve(results.map((row) => {
-        return {
+        let seller = {
           seller_id: row.key,
           name: row.name,
           directness: (row.integration_type === 'media.net') ? 'RESELLER' : 'DIRECT'
+        };
+        if (row.integration_type === 'media.net') {
+          seller.domain = 'https://www.media.net/';
```
Comment (maphe) [inline src/sellers.js]: maybe bundle the media.net changes together:
```
let seller = {
          seller_id: row.key,	
          name: row.name,	
          directness: 'DIRECT'
        };
        if (row.integration_type === 'media.net') {
          seller.domain = 'https://www.media.net/';
          seller.directness = 'RESELLER';
       }
```

Comment (epechuzal) [reply src/sellers.js]: love it, done

File: src/sellers.js [hunk 2]
```diff
@@ -16,14 +16,19 @@ const fetchPubs = async () => {
   connection.connect();
 
   const pubs = await new Promise((resolve) => {
-    connection.query(pubQuery, function (error, results, fields) {
+    connection.query(pubQuery, function (error, results) {
       if (error) throw error;
       resolve(results.map((row) => {
-        return {
+        let seller = {
```
Comment (maphe) [inline src/sellers.js]: worth trying but I believe you can have it a const as long as your not reassigning value to it.
```
const seller = ...;
seller.something = 'something else';
```
That should be allowed.

Comment (epechuzal) [reply src/sellers.js]: also done

**Comment by wsargent** [comment]
how am I seeing this?

**Pull request merged**